services/web/src/components/RequestBlock/templates/fetch.js

```diff
@@ -0,0 +1,29 @@
+import { omit } from 'lodash';
+export default function templateFetch({ url, file, ...rest }) {
+  const code = [];
+  if (file) {
+    code.push('const form = new FormData();');
+    code.push('form.append("file", fileBlob);');
+    if (rest.body) {
+      for (let [key, value] of Object.entries(rest.body || {})) {
+        code.push(
+          `form.append("${key}", ${
+            typeof value === 'string' ? `"${value}"` : value
+          })`
+        );
+      }
+    }
+    code.push('');
+    code.push(
+      `const options = ${JSON.stringify(omit(rest, ['body']), null, 2)}`
+    );
+    code.push('options.body = form;');
+    code.push('');
+  } else {
+    code.push(`const options = ${JSON.stringify(rest, null, 2)}`);
+  }
+  code.push(`fetch("${url}", options);`);
+  code.push('  .then(response => response.json())');
```

Comment:
this doesn't follow our style guide

j/k ;)

Reply:
not sure i follow

Reply:
ahh i see about using await

Reply:
no it was that our style guide/prettier would write `then((response) => response.json())`

95% a joke though :)

Reply:
also totally a bug in that code now i look at it, i will update both

Reply:
shouldn't be a bug, just a style

Reply:
ohh there was a bug :) bad semi